scripts/java-api-check-all.sh

```diff
@@ -40,7 +40,7 @@ echo "CURRENT_BRANCH: $CURRENT_BRANCH"
 echo "TARGET_BRANCH: $TARGET_BRANCH"
 readonly STABLE_RELEASE_TAG_PATTERN="v10\.[0-9]*\.0$"
 
-if [ "$TARGET_BRANCH" = "main" ]; then
+if [ "$TARGET_BRANCH" = "main" ] || [ "$TARGET_BRANCH" = "develop" ]; then
```

Comment:
Not sure why both are needed, could we just replace it when migration will happen?

Reply:
putting together a plan that we would actually need 2 protected branches, so yes we need both long term